sky/serve/service_spec.py

```diff
@@ -21,10 +21,17 @@ def __init__(
         initial_delay_seconds: int,
         min_replicas: int,
         max_replicas: Optional[int] = None,
+        target_qps_per_replica: Optional[float] = None,
+        post_data: Optional[Dict[str, Any]] = None,
+        # Deprecated: Always be True
+        auto_restart: Optional[bool] = None,
+        upscale_delay_seconds: Optional[int] = None,
+        downscale_delay_seconds: Optional[int] = None,
+        # Deprecated: the two arguments below is replaced by the
+        # target_qps_per_replica
+        # TODO(ziming): remove this in version 0.6.0
         qps_upper_threshold: Optional[float] = None,
         qps_lower_threshold: Optional[float] = None,
```

Comment:
```suggestion
        upscale_delay_seconds: Optional[int] = None,
        downscale_delay_seconds: Optional[int] = None,
        # The following arguments are deprecated.
        # TODO(ziming): remove this after 2 minor release, i.e. 0.6.0.
        # Deprecated: Always be True
        auto_restart: Optional[bool] = None,
        # Deprecated: replaced by the target_qps_per_replica.
        qps_upper_threshold: Optional[float] = None,
        qps_lower_threshold: Optional[float] = None,
```
nit: put deprecated arguments in the end of function